server/models/Request.js

```diff
@@ -0,0 +1,39 @@
+const mongoose = require("mongoose");
+const { Schema } = mongoose;
+const { ObjectId } = Schema.Types;
+const required = true;
+
+const requestSchema = new Schema({
+  user_id: {
+    type: ObjectId,
+    ref: "User",
+    required,
+  },
+  sitter_id: {
```

Comment:
We generally prefer `camelCase` for mongoose property names by convention, can we adjust these accordingly?